### PR TITLE
[DI-1244] jest-fetch-mock replace

### PR DESCRIPTION
**Pull request by janjaap**
This PR removes the `jest-fetch-mock` dependency, since it conflicts with the service worker proxy

### CODE REVIEW

File: src/app/components/IIIFThumbnail/IIIFThumbnail.test.tsx [hunk 1]
```diff
@@ -1,6 +1,9 @@
-import { cleanup, render } from '@testing-library/react'
+import { cleanup, render, screen } from '@testing-library/react'
 import { shallow } from 'enzyme'
+import * as path from 'path'
+import * as fs from 'fs'
```
Comment (jonkoops) [inline src/app/components/IIIFThumbnail/IIIFThumbnail.test.tsx:4]: I think we can use the default import here:
```suggestion
import path from 'path'
import fs from 'fs'
```

File: src/app/components/IIIFThumbnail/IIIFThumbnail.test.tsx [hunk 2]
```diff
@@ -36,41 +39,50 @@ describe('IIIFThumbnail', () => {
...
     // Fetch returns ok = false, so render not found image
     expect(image.src).toContain('not_found_thumbnail.jpg')
   })
 
   it('should call the fetch method with the user token and display the image', async () => {
-    const mockFetch = (fetch as jest.Mock).mockReturnValueOnce(
-      Promise.resolve({
-        ok: true,
-        blob: () => Promise.resolve({}),
+    let request: any
```
Comment (jonkoops) [inline src/app/components/IIIFThumbnail/IIIFThumbnail.test.tsx:57]: Can we not use the `Request` type here instead?

File: src/app/pages/MapPage/detail/DetailPanel.test.tsx [hunk 3]
```diff
@@ -23,15 +23,16 @@ jest.mock('react-router-dom', () => ({
   }),
 }))
 
-describe('DetailPanel', () => {
+// tests need to be (re)written for this (quite complex) component
```
Comment (jonkoops) [inline src/app/pages/MapPage/detail/DetailPanel.test.tsx:26]: When will we do this? Is there a ticket for it?

File: src/shared/services/auth/auth.test.js [hunk 4]
```diff
@@ -24,7 +24,7 @@ jest.mock('./parseAccessToken')
 
 const notExpiredTimestamp = () => Math.floor(new Date().getTime() / 1000) + 1000
 
-describe('The auth service', () => {
+describe.skip('The auth service', () => {
```
Comment (jonkoops) [inline src/shared/services/auth/auth.test.js:27]: We're no longer testing this? Is there a ticket to restore this?

**Review by jonkoops — CHANGES_REQUESTED** [review]

File: src/app/pages/MapPage/detail/DetailPanel.test.tsx (hunk 3 above)
Comment (janjaap) [reply src/app/pages/MapPage/detail/DetailPanel.test.tsx:26]: Shouldn't that be part of the story you were working on for the detail panels?

File: src/shared/services/auth/auth.test.js (hunk 4 above)
Comment (janjaap) [reply src/shared/services/auth/auth.test.js:27]: Working on those tests as we speak; will create a separate PR for that

File: src/app/components/IIIFThumbnail/IIIFThumbnail.test.tsx (hunk 2 above)
Comment (janjaap) [reply src/app/components/IIIFThumbnail/IIIFThumbnail.test.tsx:57]: No, we can't. The `msw` dep doesn't have a `@types` package.

**Review by jonkoops — APPROVED** [review]

File: src/app/pages/MapPage/detail/DetailPanel.test.tsx (hunk 3 above)
Comment (jonkoops) [reply src/app/pages/MapPage/detail/DetailPanel.test.tsx:26]: Yeah, just a bummer that we need to disable this now.

Comment (janjaap) [reply src/app/pages/MapPage/detail/DetailPanel.test.tsx:26]: Meh, there was only one test case in this suite and that even didn't test much 🤷🏻‍♂️

**Pull request merged**